task2/index.js

```diff
@@ -0,0 +1,25 @@
+function storeDetails(){  
+    var name=document.getElementById("username").value;  
+    var password=document.getElementById("password").value;  
+    var lowerCaseLetters = /[a-z]/g;
+    var upperCaseLetters = /[A-Z]/g;
+    var numbers = /[0-9]/g;
+    if(name.length == 0 || password.length == 0 || password.length<8){
+        // !password.match(numbers)|| !password.match(upperCaseLetters) || !password.match(lowerCaseLetters)){
+        window.location.href="./index.html";
+        window.location.reload(true)
+        window.location.replace("./index.html")
+        alert("please fill in email & password correctly i.e., password must be of atleast 8 characters long including uppercase,lowercase letter and 1 number");
+        return
+       
+    }else{
+        localStorage.setItem("name", name);
+        localStorage.setItem("password", password);
```

Comment:
Instead of setting/removing multiple items on localStorage, we can create an object and save it in stringified form. Try searching 'storing arrays or objects to local storage' and implement the same wherever required.

@shubhamshukla91

Reply:
yeah sure,
const user = {
        name: this.name,
       password: this.password,
        }    

   var userInput = [];
   userInput.push(user);

   localStorage.setItem('user', JSON.stringify(userInput));

Reply:
I'll surely update it.